src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +70,22 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+    public Block getBlock() {
+        return block;
+    }
+
+    public Street getStreet() {
+        return street;
+    }
+
+    public Unit getUnit() {
+        return unit;
+    }
+
+    public PostalCode getPostalCode() {
+        return postalCode;
+    }
+    
```

Comment:
I think you missed out the updating of `toString`, `hashCode` and `equals` methods.